examples/kubeflow/mnist_with_tfjob/README.md

```diff
@@ -0,0 +1,194 @@
+
+#### Mnist. Mnist. Mnist.  
+
+Once our code trains the model, it needs to save it somewhere. Just like we copied data into the container we can copy it back out again, all the while still maintaining some provenance. If you take a look at the `tfjob_mist.py` and scroll towards the bottom you'll see that we're just crawling the same `/tmp/data/` directory and copying it to the Pachyderm S3 Gateway output `url:s3://<pachyderminstance>/master.outputrepo:/data/`
+
+
+```
+print("walking {} for copying to {}".format(args.datadir, output_url))
+  for dirpath, dirs, files in os.walk(args.datadir, topdown=True):   
+    for file in files:
+      uri = os.path.join(dirpath, file)
+      newpath = output_url + file
+      print("copying {} to {}".format(uri, newpath))
+      file_io.copy(uri, newpath, True)
+```
+
+Next, let's move onto how we deploy our code. Start by taking a look at the `tf_job_s3_gateway.yaml`:
+
+```
+apiVersion: "kubeflow.org/v1beta2"
+kind: "TFJob"
+metadata:
+  name: "mnist-pach-s3-gateway-example"
+  namespace: kubeflow 
+spec:
+  cleanPodPolicy: None 
+  tfReplicaSpecs:
+    Worker:
+      replicas: 1 
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - name: tensorflow
+              image: nickharvey/pach_kflow_example:v1
+              env:
+                # This endpoint assumes that the pachd service was deployed
+                # in the namespace pachyderm.
+                # You may replace this with pachd.<namespace> if you deployed
+                # pachyderm in another namespace. For example, if deployed
+                # in default it would be pachd.default. You may also
+                # hard code in the pachd CLUSTER-IP address you obtain from
+                # kubectl get services -n <namespace>
+                - name: S3_ENDPOINT
+                  value: "pachd.pachyderm:600"
+                - name: S3_USE_HTTPS
+                  value: "0"
+                - name: S3_VERIFY_SSL
+                  value: "0"
+              command:
+                - "python3"
+                - "tfjob.py"
+                - "-i"
+                - "master.inputrepo"
+                - "-o"
+                - "master.outputrepo"
+                - "-d"
+                - "/tmp/data/"
+```
+The 'mnist_tf_job_s3_gateway.yaml' is our spec file that Kubeflow and Kubernetes will use to deploy our code. You can find out everything you need to know about this spec file in the [Kubeflow TFjobs Docs](https://www.kubeflow.org/docs/components/training/tftraining/). Notice the Pachyderm repos, branches, and tmp location are being declared at the bottom.  
+
+To deploy just run the following:
```

Comment:
To deploy the TF job manifest, run the following command:

```
kubectl create -f mnist_tf_job_s3_gateway.yaml
```

Can you please use "```" with all command blocks instead of "`".